src/gradata/hooks/agent_graduation.py

```
@@ -0,0 +1,53 @@
+        emit(
+            "AGENT_OUTCOME",
+            source="hook:agent_graduation",
+            data={
+                "agent_type": agent_type,
+                "output_preview": preview,
+                "output_length": len(output),
+            },
+            brain_dir=brain_dir,
+        )
+    except Exception:
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)